gui/WorldView.cxx

```diff
@@ -1811,10 +1812,7 @@ void WorldView::saveFusedMeshFrameColors(const QString &path, bool occlusion)
   coloration->set_frame_sampling(d->volumeOptions->getFrameSampling());
   double occlusionThreshold = d->volumeOptions->getOcclusionThreshold();
   coloration->set_occlusion_threshold(occlusionThreshold);
-  coloration->set_remove_occluded(occlusion);
-  vtkSmartPointer<vtkPolyData> meshFrameColors = vtkSmartPointer<vtkPolyData>::New();
```

Comment:
Why are these lines removed?

Reply:
`coloration->set_output()` was replaced with `get_output()`, and as far as I can tell, the only purpose of the `set_output()` was to provide a polydata for the results, which is now handled internally. I haven't examined the changed code in kwiver, though - the api changes came from https://github.com/Kitware/kwiver/pull/1534

Reply:
yep, confirmed that creating the output polydata is handled internally using a ShallowCopy of the input, or other vtk pipeline result. https://github.com/Kitware/kwiver/pull/1534/files#diff-a4e99234c2dc9ae926db06dca4333e8dd8c49958ad50879ca8be3f45d1311794R215